### PR TITLE
Bug 1250681 - Migrate remaining TV apps to l20n

### PR DESCRIPTION
**Pull request by zbraniecki**
https://bugzilla.mozilla.org/show_bug.cgi?id=1250681

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a99885f46c9dfd2d1e33aa946b1ef86e00ffaad3

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=11adb236a1a05ba2e26ffdbfc7ae404e23e71fe2

File: tv_apps/app-deck/bower_components/smart-modal-dialog/base-modal-dialog.js [hunk 1]
```diff
@@ -53,8 +53,8 @@
   proto._setL10n = function(element, l10n) {
     if ((typeof l10n) === 'string') {
       element.setAttribute('data-l10n-id', l10n);
-    } else if (navigator.mozL10n) {
-      navigator.mozL10n.setAttributes(element, l10n.id, l10n.args);
```
Comment (rexboy7) [inline tv_apps/app-deck/bower_components/smart-modal-dialog/base-modal-dialog.js]: It's ok to just land it as is, but we need to update it to `fxos-components` too. You can open a bug for it.
`smart-modal-dialog` is now renamed to `fxos-tv-modal-dialog`. And it should replace smart-modal-dialog recently.
https://github.com/fxos-components/fxos-tv-modal-dialog

Comment (zbraniecki) [inline tv_apps/app-deck/bower_components/smart-modal-dialog/base-modal-dialog.js]: Filed as bug 1252791

**Pull request merged**